rfc/rfc003-oauth2-authorization.md

```diff
@@ -227,7 +227,7 @@ The **usi** field in the JWT contains the signed login contract. If present it M
 
 **5.2.1.6 Endpoint validation**
 
-The **aud** field MUST match the identifier of the registered endpoint. This prevents the use of the JWT at any other endpoint. The endpoint reference is used for this. \([RFC7523](https://tools.ietf.org/html/rfc7523#section-3)\). An endpoint identifier contains a DID and a fragment.
+The **aud** field MUST match the URL of the registered oauth endpoint. This prevents the use of the JWT at any other endpoint. The concrete endpoint URL is used for this. \([RFC7523](https://tools.ietf.org/html/rfc7523#section-3)\). The `purposeOfUse` field identifies the service/Bolt which in turn refers to the correct endpoint URL. 
```

Comment:
"oauth" as in the compound service? Than I'd say `oauth` (with backticks to indicate it's a field). "oauth" as in the spec? Then I'd capitalize it propertly; OAuth.

Reply:
changed the last part of the sentence